redhat-appstudio/service-provider-integration-oauth
# update to benefit from the token refresh implemented in the operator

### What does this PR do?
A companion PR to https://github.com/redhat-appstudio/service-provider-integration-operator/pull/264 to benefit from the token refresh functionality also in the oauth service.

### What issues does this PR fix or reference?
NA

### How to test this PR?
see https://github.com/redhat-appstudio/service-provider-integration-operator/pull/264
